com.scoutshonour.dtipbijays.yaml

```diff
@@ -0,0 +1,46 @@
+app-id: com.scoutshonour.dtipbijays
+runtime: org.freedesktop.Platform
+runtime-version: 1.6
+sdk: org.freedesktop.Sdk
+command: don't take it personally, babe.sh
```

Comment:
Shouldn't it point at dtipbijays-launcher.sh?

Reply:
Hi, that name is the name of the original launch script.
I replace it with a modified one which starts on the /app folder, instead of letting it try to guess based on the current dir.

See this line:
` - install -Dm755 dtipbijays-launcher.sh /app/bin/"don't take it personally, babe.sh"`

Reply:
Right, oversight on my side. :+1: